tgui/packages/tgui/interfaces/pda/pda_mob_hunt.js

```diff
@@ -0,0 +1,132 @@
+import { useBackend } from "../../backend";
+import { LabeledList, Box, Button, Section, Flex } from "../../components";
+
+export const pda_mob_hunt = (props, context) => {
+  const { act, data } = useBackend(context);
+
+  const {
+    connected,
+    wild_captures,
+    no_collection,
+    entry,
+  } = data;
+
+  return (
+    <Box>
+      <LabeledList>
+        <LabeledList.Item label="Connection Status">
+          {connected ? (
+            <Box color="green">
+              Connected
+              <Button
+                ml={2}
+                content="Disconnect"
+                icon="sign-out-alt"
+                onClick={() => act('Disconnect')}
+              />
+            </Box>
+          ) : (
+            <Box color="red">
+              Disconnected
+              <Button
+                ml={2}
+                content="Connect"
+                icon="sign-in-alt"
+                onClick={() => act('Reconnect')}
+              />
+            </Box>
+          )}
+        </LabeledList.Item>
+        <LabeledList.Item label="Total Wild Captures">
+          {wild_captures}
+        </LabeledList.Item>
+      </LabeledList>
+      <Section
+        title="Collection"
+        mt={2}
+        buttons={
+          // The double box wrapping is imporant here
+          // Eslint argues with itself over indentation
+          <Box>
+            {!no_collection && (
+              <Box>
+                <Button
+                  content="Previous"
+                  icon="arrow-left"
+                  onClick={() => act('Prev')} />
+                <Button
+                  content="Next"
+                  icon="arrow-right"
+                  onClick={() => act('Next')} />
+              </Box>
+            )}
+          </Box>
+        }>
+        {no_collection ? (
```

Comment:
Do you need {!!no_collection here?

Reply:
Nope, its a ternary `?` not a singular check `&&`